frontend/src/components/applications/StaffingCertificateInput.vue

```diff
@@ -32,7 +32,7 @@
               @input="update(certificate)"
               @blur="checkDuplicateCertificateNumbers" />
           </v-col>
-          <v-col v-if="certificate.initials || certificate.certificateNumber" cols="12" sm="1">
+          <v-col v-if="!readonly && (certificate.initials || certificate.certificateNumber)" cols="12" sm="1">
```

Comment:
Given that there's three conditions here I'd be inclined to move this to a method. E.g. showTrash().

Reply:
it's updated.